src/main/java/org/kohsuke/github/GHWorkflowRunJobQueryBuilder.java

```diff
@@ -0,0 +1,47 @@
+package org.kohsuke.github;
+
+import java.net.MalformedURLException;
+
+/**
+ * Lists up jobs of a workflow run with some filtering.
+ *
+ * @author Guillaume Smet
+ */
+public class GHWorkflowRunJobQueryBuilder extends GHQueryBuilder<GHWorkflowRunJob> {
```

Comment:
Even if there's not really search filters, I decided to go with this pattern to be consistent with the work I started (but not yet pushed) on the check runs. They are implemented in the exact same way (so with this weird `filter` thing) but with more criteria.

Reply:
I kept this class but I pushed an additional commit to hide it from the users as for now it doesn't bring anything valuable.

Reply:
I'm all in favor of less API surface area.